app/models/pydantic/jobs.py

```diff
@@ -130,7 +130,7 @@ class PixETLJob(Job):
     vcpus = MAX_CORES
     memory = MAX_MEM
     num_processes = max(int(MAX_CORES * 2 / 3), 1)
-    attempts = 4
+    attempts = 10
```

Comment:
Is this supposed to be part of your PR? Seems unrelated

Reply:
As you figured in other comment there was a mismatch in the base of these two branches: `metadata-final` was based on `develop` and `metadata-endpoints-crud` was based on master which had a lot of hot fixes. I have fixed that now.